Npm depdency doesn't exist

Our ci is failing on the error:

> npm ERR! code ETARGET
> npm ERR! notarget No matching version found for ecstatic@^3.0.0
> npm ERR! notarget In most cases you or one of your dependencies are requesting
> npm ERR! notarget a package version that doesn't exist.
> npm ERR! notarget 
> npm ERR! notarget It was specified as a dependency of 'http-server'
> npm ERR! notarget 

Looks like the issue is going to be resolved any minute as soon as this pr is merged: https://github.com/jfhbrook/node-ecstatic/pull/256

--- a/package.json
+++ b/package.json
@@ -33,7 +33,7 @@
     "concurrently": "^4.1.0",
     "elm-format": "^0.8.1",
     "elm-live": "^3.4.0",
-    "elm-test": "^0.19.0-rev6",
+    "elm-test": "^0.19.0",
     "http-server": "^0.11.1",
     "nodemon": "^1.18.10",
     "watchify": "^3.11.1"

